# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/__P.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/md5.c.vanilla	2019-01-27 17:34:20.878960396 +0000
+++ mine-0.23/md5.c	2019-01-27 17:36:23.926958994 +0000
@@ -39,6 +39,10 @@
 # define SWAP(n) (n)
 #endif
 
+#ifndef __P
+#define __P(x) x
+#endif
+
 typedef unsigned int md5_uint32;
 
 /* Structure to save state of computation between the single steps.  */
